project/frontend_views/views.py

```diff
@@ -266,3 +268,18 @@ def about_view(request):
 
 def support_us_view(request):
     return TemplateResponse(request, 'static_templates/support_us.html', {})
+
+@csrf_exempt
+def civi2csv(request, thread_id):
```

Comment:
Add a [docstring](https://www.python.org/dev/peps/pep-0257/#what-is-a-docstring) to this function, describing its overall purpose.